configs/stage/roles/hybrid-committed-spend.json

```diff
@@ -0,0 +1,17 @@
+{
+  "roles": [
+    {
+      "name": "Hybrid Committed Spend viewer",
+      "description": "View any Hybrid Committed Spend report.",
+      "system": false,
```

Comment:
@lorenzobenvenuti do you want non-system role ? 
- it could be deleted and updated - preference here is to have system role which is seeded.

Note: I found that we are setting to [True](https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/role/definer.py#L71) by default during seeding. So maybe it will be better to set it to true.

Reply:
Thanks @lpichler , I overlooked that field. Updated to `true`.

Thanks,

lorenzo